cmd/autoupdate/main.go

```diff
@@ -87,9 +100,15 @@ func main() {
 			}
 		}
 
+		if pckg.Autoupdate.Source == "git" {
+			util.Debugf(ctx, "running git update")
+			newVersionsToCommit, latestVersion = updateGit(ctx, pckg)
+		}
```

Comment:
is this duplicated now?

Reply:
thanks, good call. Didn't notice when I merged